src/hooks/UrqlUseSubscriptionWithHandler.re

```diff
@@ -1,38 +1,42 @@
+type handler('acc, 'resp, 'ret) =
```

Comment:
Let's also make sure to rename this module to `UrqlUseSubscription.re` and get rid of the old `UrqlUseSubscription.re` before merging 😄

Reply:
I was thinking of only doing this as the last step, so that the line diff doesn't become too big to review

For now I'm not reexporting `UrqlUseSubscription` instead of deleting it right away.

Do you think it's better to just delete it now?

Reply:
Good point, let's do it as the last commit after approval 👍